content/includes/veteran-navigation.html

```diff
@@ -70,7 +70,7 @@
                 <li><a href="/education/apply/">Application process</a></li>
                 <li><a href="/education/gi-bill/">GI Bill programs</a></li>
                 <li><a href="/gi-bill-comparison-tool/">Compare GI Bill benefits</a></li>
-                <li><a href="/education/apply-for-education-benefits/application/1990" class="usa-button va-button-primary">Apply Now</a></li>
+                <li><a href="/education/apply/" class="usa-button va-button-primary">Apply Now</a></li>
```

Comment:
Now both the `Application process` link and the `Apply Now` button go to apply page. If that's what we want for clarity, great, otherwise we can get rid of one.

@goldenmeanie Is that a question for you?